src/libraries/Microsoft.Extensions.DependencyModel/src/DependencyContextPaths.cs

```diff
@@ -42,7 +40,13 @@ private static DependencyContextPaths GetCurrent()
 
         internal static DependencyContextPaths Create(string? depsFiles, string? sharedRuntime)
```

Comment:
Can this be private?

Reply:
No, unit tests use it through IVT. They could be refactored to use reflection instead, but I'm trying to get this through to unblock code flow - see https://github.com/dotnet/installer/pull/16621#issuecomment-1587705679.

Reply:
Oh IVT, how you mock me.